systemtests/tests/multiple-clients/etc/bareos/bareos-dir.d/fileset/SelfTest.conf.in

```diff
@@ -0,0 +1,11 @@
+FileSet {
+  Name = "SelfTest"
+  Description = "fileset just to backup some files for selftest"
+  Include {
+    Options {
+      Signature = MD5 # calculate md5 checksum per file
+    }
+   #File = "@sbindir@"
```

Comment:
A lot of tests have this and i am not sure why.  I would remove it on new tests.